### PR TITLE
Platformtests blueprints

### PR DESCRIPTION
**Pull request by Vladimir-Antonovich**

### CODE REVIEW

File: tests/platformtests/resources/bind_policy_group/blueprint.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This Blueprint installs VM via vsphere-plugin and create and set security tag
+  via NSX plugin on an vsphere environment.
+
```
Comment (0lvin) [inline tests/platformtests/resources/bind_policy_group/blueprint.yaml]: vSphere plugin is not included?

File: tests/platformtests/resources/security_policy/blueprint.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This Blueprint installs VM via vsphere-plugin and create and set security tag
+  via NSX plugin on an vsphere environment.
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.4/types.yaml
+#  - http://www.getcloudify.org/spec/vsphere-plugin/2.1.0/plugin.yaml
```
Comment (0lvin) [inline tests/platformtests/resources/security_policy/blueprint.yaml]: Vsphere is not included?

File: tests/platformtests/test_security_tag.py [hunk 3]
```diff
@@ -128,6 +128,72 @@ def test_securitytag(self):
...
+        """Deploying security tag minimal test"""
+
+        # set blueprint name
+        blueprint = os.path.join(
+            self.blueprints_path,
+            'security_tag/blueprint.yaml'
+        )
+
+        #set inputs name
+        inputs = os.path.join(
```
Comment (0lvin) [inline tests/platformtests/test_security_tag.py]: We will never have credentials in input files for tests, so use os.environ.get('*') for now.

**Comment by Vladimir-Antonovich** [comment]
New changes will be pulled

**Pull request closed without merging**